### PR TITLE
Added Session Termination UseCase.

### PR DESCRIPTION
**Pull request by ujji8055**
## Related Issues
Terminate User's SSO sessions.

## Description
Terminate all the user's SSO sessions on receiving **Account Lockout Alert**.

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py
Comment (daryakoval) [inline Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py:571]: Is there an option to get to page number 3 using xsoar command?

File: Packs/SafeNet_Trusted_Access/IncidentTypes/incidenttype-SafeNet_Trusted_Access_Account_Lockout_Alert.json
Comment (daryakoval) [inline Packs/SafeNet_Trusted_Access/IncidentTypes/incidenttype-SafeNet_Trusted_Access_Account_Lockout_Alert.json:27]: If there is no specific reason for this incident type to has fromvesion 5.5.0, please change to 6.0.0.
```suggestion
    "fromVersion": "6.0.0"
```

File: Packs/SafeNet_Trusted_Access/Classifiers/classifier-SafeNet_Trusted_Access_–_Account_Lockout_Alert_Classifier.json
Comment (daryakoval) [inline Packs/SafeNet_Trusted_Access/Classifiers/classifier-SafeNet_Trusted_Access_–_Account_Lockout_Alert_Classifier.json:4]:
```suggestion
    "id": "SafeNet Trusted Access – Account Lockout Alert Classifier",
```

File: Packs/SafeNet_Trusted_Access/.pack-ignore
Comment (daryakoval) [inline Packs/SafeNet_Trusted_Access/.pack-ignore:2]: Why you are ignoring this error?

File: Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md:3]: Done. Updated as shown below.

image

File: Packs/SafeNet_Trusted_Access/README.md
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/README.md:7]: Done. Updated as shown below.

image

File: Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.yml
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.yml:716]: No, API doesn't have an offset parameter. 

image

File: Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/README.md
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/README.md:158]: Done. Updated for every occurance.

image

File: Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.yml
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.yml:765]: By default (without limit parameter), it'll fetch information of the first page(index-0).
No default value for the limit. The maximum value is 10000. Validation is present in the code for the same.

File: Packs/SafeNet_Trusted_Access/Playbooks/playbook-SafeNet_Trusted_Access_-_Add_to_Unusual_Activity_Group.yml
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/Playbooks/playbook-SafeNet_Trusted_Access_-_Add_to_Unusual_Activity_Group.yml:8]: This is suggested by the bussiness. As we have recently uploaded our previous versions and have less number of downloads. So it's ok.

File: Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py:571]: No such parameter/argument for the commands to change the page no.

File: Packs/SafeNet_Trusted_Access/IncidentTypes/incidenttype-SafeNet_Trusted_Access_Account_Lockout_Alert.json
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/IncidentTypes/incidenttype-SafeNet_Trusted_Access_Account_Lockout_Alert.json:27]: Done.

image

File: Packs/SafeNet_Trusted_Access/Classifiers/classifier-SafeNet_Trusted_Access_–_Account_Lockout_Alert_Classifier.json
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/Classifiers/classifier-SafeNet_Trusted_Access_–_Account_Lockout_Alert_Classifier.json:4]: Done. Updated ID to classifier name.

image

File: Packs/SafeNet_Trusted_Access/.pack-ignore
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/.pack-ignore:2]: As discussed with Kobbi, this is affecting the usecase functionality. So as suggested, added this under the .pack-ignore file.
For our previous version pack, I am not able to see our playbook readme like https://xsoar.pan.dev/docs/reference/playbooks/abuse-inbox-management-detect--respond 
Please check and let us know the exact reason for the same.

Comment (kgal-pan) [reply Packs/SafeNet_Trusted_Access/.pack-ignore:2]: This error indicates that the Playbook is bound to a specific Integration instance. I informed Joshi that this is not best practice (that the Integration instance name is used as an input to a Playbook). 

Joshi informed me that they have a guide to that instructs users to specify the instance name.

Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/.pack-ignore:2]: Thank you Kobbi.

File: Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md
Comment (daryakoval) [inline Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md:27]:
```suggestion
- ***Breaking Change:*** Changed playbook inputs names. 
```

**Review by daryakoval — COMMENTED** [review]
Please run `demisto-sdk update-release-notes -i Packs/<Pack Name> -bc`, 
to add breaking change note, do not forget to change `breakingChangesNotes` field in the new created json file, as explained [here](https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version).

File: Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md
Comment (ujji8055) [reply Packs/SafeNet_Trusted_Access/ReleaseNotes/2_0_0.md]: Done. As discussed, updated release note and added the **.json** file.

**Comment by daryakoval** [comment]
Hi, you can convert timestamp to datastring in the human readable of sta-get-user-sessions command, using `timestamp_to_datestring` [function](https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L6443). 
***sta-get-user-application*** - check why this command not working.

**Comment by ujji8055** [comment]
@daryakoval 
The command is working as expected. As discussed within the team, there was some issue with our test environment Swagger Rest API. Please find the screenshot below.

image

**Comment by ujji8055** [comment]
@daryakoval 
Please put the pack on hold for now. I am incorporating Elia comments. I will let you know once it's good to merge.

**Comment by ujji8055** [comment]
@evisochek - I have created a common classifier and mapper for both the use-case. As discussed with Kobbi, added Deprecated into old classifier files and updated the incident type name for every occurrence.
Kindly review and suggest the needful.

**Review by daryakoval — APPROVED** [review]

**Pull request merged**
